worker_tasks/scraper.py

```diff
@@ -74,6 +77,10 @@ def scrape_articles(self):
         for feed in Feed.query.all():
             self.store_items_and_category_counts(feed)
 
-        # TODO: Assign categories to articles
-
-
+        # Assign categories to articles after all feeds are processed
+        for item in Item.query.filter_by(category_1=None).all():
+            (cat1, cat2) = categorizeItem(item.title, item.text, self.title_categories)
+            item.category_1 = cat1
+            item.category_2 = cat2
+            db.session.commit()
+            logging.info("Categorized item %s: %s, %s" % (item.title, item.category_1, item.category_2))
```

Comment:
Just to make logging easier to read among the stream of text, can we rewrite this to something like:

CATEGORIZED [%s]: (%s, %s)